packages/create-angular/src/app/components/widgets/FormulaWidget.component.ts

```diff
@@ -64,7 +71,7 @@ export class FormulaWidgetComponent {
         )
         .then((response) => {
```

Comment:
Does it make sense to type the `response` instead?

Reply:
The `response` type is known to be FormulaResponse, unfortunately we need the cast because the officially the response type can be `null`. I'm not sure it should be _nullable_ so I have a TODO to try to change the type in the API client, rather than trying to do special null-handling in the template for now. :)

Reply:
Gotcha! Yeah, we should probably be more explicit with the `response`, otherwise casting it as a consumer feels a bit strange 👍